webpack.config.ts

```diff
@@ -5,6 +5,11 @@ import { ConsoleRemotePlugin } from '@openshift-console/dynamic-plugin-sdk-webpa
 import * as CopyWebpackPlugin from 'copy-webpack-plugin';
 import * as webpack from 'webpack';
 
+const resolveLocale = (dirName: string, ns: string) =>
+  ['en', 'ja', 'ko', 'zh'].reduce((acc, lang) => ([...acc, { from: path.resolve(dirName, `locales/${lang}/plugin__*.json`), to: `locales/${lang}/${ns}.[ext]`}]), [])
```

Comment:
Create a constant for the languages used:
export const LANGUAGES =  ['en', 'ja', 'ko', 'zh'];
```suggestion
 LANGUAGES.reduce((acc, lang) => ([...acc, { from: path.resolve(dirName, `locales/${lang}/plugin__*.json`), to: `locales/${lang}/${ns}.[ext]`}]), [])
```